modules/nf-core/pbmm2/align/meta.yml

```diff
@@ -0,0 +1,61 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "pbmm2_align"
+description: Alignment with PacBio's minimap2 frontend
+keywords:
+  - sort
+  - example
+  - genomics
+tools:
+  - "pbmm2":
+      description: "A minimap2 frontend for PacBio native data formats"
+      homepage: "https://github.com/PacificBiosciences/pbmm2"
+      documentation: "https://github.com/PacificBiosciences/pbmm2"
+      tool_dev_url: "https://github.com/PacificBiosciences/pbmm2"
+      licence: ["BSD-3-clause-Clear"]
+      identifier: biotools:pbmm2
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - bam:
+        type: file
+        description: Sorted BAM/CRAM/SAM file
+        pattern: "*.{bam,cram,sam}"
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - fasta:
+        type: file
+        description: Sorted BAM/CRAM/SAM file
+        pattern: "*.{fasta}"
+output:
+  - bam:
+      - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*.bam":
+          type: file
+          description: Sorted BAM/CRAM/SAM file
+          pattern: "*.{bam,cram,sam}"
+          ontologies:
+            - edam: "http://edamontology.org/format_25722"
+            - edam: "http://edamontology.org/format_2573"
+            - edam: "http://edamontology.org/format_3462"
```

Comment:
What is this? Never seen before.

Reply:
Never mind, I guess that this was added to the template when you create a new module? But the first link is broken, could you remove the last "2" in `http://edamontology.org/format_25722`?

Reply:
Yeah - it was added in the template, so I kept it - will do!